openslides_backend/action/actions/poll/update.py

```diff
@@ -96,3 +96,19 @@ def check_100_percent_base(self, instance: Dict[str, Any]) -> None:
             )
             pollmethod = poll.get("pollmethod")
         base_check_100_percent_base(pollmethod, onehundred_percent_base)
+
+
+def check_state_change(instance: Dict[str, Any], poll: Dict[str, Any]) -> bool:
```

Comment:
Add this function as a member to the class.

Reply:
Done.